redirects.yaml

```
@@ -42,6 +42,34 @@ redirects:
 - from: /docs/webstore/pricing/
   to: /docs/webstore/money/
 
+# CWS payments/apps purge
+- from: /docs/webstore/hosted_apps
+  to: /docs/extensions/webstore
+- from: /docs/webstore/migrating
+  to: /docs/extensions/webstore
+- from: /docs/webstore/get_started_simple
+  to: /docs/extensions/webstore/publish/
+- from: /docs/webstore/apps_vs_extensions
+  to: /docs/extensions/webstore
+- from: /docs/webstore/distribution
+  to: /docs/extensions/webstore
+- from: /docs/webstore/launching
+  to: /docs/extensions/webstore
+- from: /docs/webstore/choosing
+  to: /docs/extensions/webstore
+- from: /docs/webstore/samples
+  to: /docs/extensions/webstore
+- from: /docs/webstore/get_started
+  to: /docs/extensions/webstore
+- from: /docs/webstore/inline_installation
+  to: /docs/extensions/webstore
```

```suggestion
- from: /docs/webstore/hosted_apps
  to: /docs/webstore
- from: /docs/webstore/migrating
  to: /docs/webstore
- from: /docs/webstore/get_started_simple
  to: /docs/webstore/publish/
- from: /docs/webstore/apps_vs_extensions
  to: /docs/webstore
- from: /docs/webstore/distribution
  to: /docs/webstore
- from: /docs/webstore/launching
  to: /docs/webstore
- from: /docs/webstore/choosing
  to: /docs/webstore
- from: /docs/webstore/samples
  to: /docs/webstore
- from: /docs/webstore/get_started
  to: /docs/webstore
- from: /docs/webstore/inline_installation
  to: /docs/webstore
```

`/docs/extensions/webstore` does not exist